### PR TITLE
Release 2.2.0

### PR DESCRIPTION
**Pull request by alimanfoo**
This PR provides the 2.2.0 release. Also includes merged in changes from the rc branch that were made for the 2.2.0rc releases.

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -12,29 +12,29 @@ source:
...
     - python
-    - cython
     - setuptools >18.0
     - setuptools_scm >1.5.4
   run:
     - python
+    - asciitree
     - numpy >=1.7
     - fasteners
+    - numcodecs >=0.5.3
```
Comment (jakirkham) [inline recipe/meta.yaml]: Should be 0.5.4, right?

Comment (jakirkham) [reply recipe/meta.yaml]: NVM guess we still have it as 0.5.3 in Zarr's `setup.py` and PyPI. Sorry for missing that.

Comment (alimanfoo) [reply recipe/meta.yaml]: No problem, we require >=0.5.3 because that's when the nosetest dependency was removed. Both 0.5.3 and 0.5.4 should work.

**Comment by jakirkham** [comment]
Just realized this was under the `rc` label. Sorry, should have caught that. Fixing with PR ( https://github.com/conda-forge/zarr-feedstock/pull/17 ).

**Pull request merged**